point/nexstar.py

```diff
@@ -398,6 +399,65 @@ def set_time(self, timestamp=None):
         ])
         self._send_command(command)
 
+    def get_gps_lock_status(self):
+        """Get the status of GPS lock
+
+        Returns:
+            bool: True if GPS is linked (locked?), false if GPS is not linked (no lock?)
+        """
+        command = b'P' + bytes([
+            1,
+            176,  # GPS Device ID?
+            55,  # Device register?
+            0,
+            0,
+            0,
+            1,  # GPS response bytes?
+        ])
+
+        response = self._send_command(command, 1)
+
+        return bool(response[0])
+
+    def get_gps_location(self):
+        """Get the GPS mount location on Earth in geographic (latitude/longitude) coordinates.
+
+        Returns:
+            tuple of floats: A pair of angles (latitude, longitude) in signed degrees format.
+        """
+        [x_var, y_var, z_var] = self._send_command(b'P' + bytes([1, 176, 1, 0, 0, 0, 3]), 3)
+        lat = ((x_var * 65536) + (y_var * 256) + z_var) / (2 ** 24) * 360
+
+        [x_var, y_var, z_var] = self._send_command(b'P' + bytes([1, 176, 2, 0, 0, 0, 3]), 3)
+        lon = ((x_var * 65536) + (y_var * 256) + z_var) / (2 ** 24) * 360
+
+        return (lat, lon)
+
+    def get_gps_time(self):
+        """Get the current GPS time from the hand controller in seconds since the Unix epoch.
```

Comment:
I think strictly speaking this command is getting time directly from the GPS device rather than the hand controller. I think for all of the 'P' commands the hand controller is just a middle man.

Reply:
Agree, will update docstrings.